vhdbuilder/packer/init-variables.sh

```diff
@@ -352,6 +352,8 @@ if [ "$OS_TYPE" == "Windows" ]; then
   if [ "null" != "${OS_DISK_SIZE}" ]; then
     echo "Setting os_disk_size_gb to the value in windows-settings.json for ${WINDOWS_SKU}: ${OS_DISK_SIZE}"
     os_disk_size_gb=${OS_DISK_SIZE}
+  else
+    os_disk_size_gb="30"
```

Comment:
Could we throw an error if OS_DISK_SIZE is not set and set OS_DISK_SIZE for WS2019 in https://github.com/Azure/AgentBaker/blob/45394125976b8acc748602665eb913382cfa1d80/vhdbuilder/packer/windows/windows_settings.json#L23?

Reply:
I'll do that as well - but it's good to have a fallback.

Reply:
Have set it to 30GB and also changed the schema validator for windows_settings.json to fail if os_disk_size is unset.